config/crd/bases/actions.summerwind.dev_runnerdeployments.yaml

```diff
@@ -404,20 +404,12 @@ spec:
                       properties:
                         limits:
                           additionalProperties:
-                            anyOf:
-                              - type: integer
-                              - type: string
-                            pattern: ^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?$
-                            x-kubernetes-int-or-string: true
+                            type: string
```

Comment:
Perhaps this regression(?) disappear once you upgrade your `controller-gen` binary. 0.3.0 should be automatically installed via [our make target](https://github.com/summerwind/actions-runner-controller/blob/eb002cfed45b8598b1ff825e2fe7020118f4346c/Makefile#L148-L165), after you `rm $GOBIN/bin/controller-gen` or `rm ~/go/bin/controller-gen`.

Reply:
I was on an old version, it did not update automatically (because it could still find a binary). Once I removed it, the proper controller-gen version was installed automatically. So, works!